code-buddies.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/index.css" />
+    <title>Student Companion App</title>
+  </head>
+  <body>
+    <header class="header">
+      <h1 class="header__title">Code Buddys</h1>
+      <span class="header__subtitle">on Monday - 25.01.2021</span>
```

Comment:
Maybe a `h2`-element instead as the class name suggests that it is a headline?
```suggestion
      <h2 class="header__subtitle">on Monday - 25.01.2021</h2>
```